docs/reference/aggregations/bucket/composite-aggregation.asciidoc

```diff
@@ -353,6 +353,77 @@ include::datehistogram-aggregation.asciidoc[tag=offset-result-intro]
 
 include::datehistogram-aggregation.asciidoc[tag=offset-note]
 
+[[_geotile_grid]]
+===== GeoTile Grid
+
+Works on `geo_point` fields and groups points into buckets that represent
```

Comment:
This page is so long I don't think we should leave sentences incomplete and assume they've read all the preceding info:
```suggestion
The `geotile_grid` value source works on `geo_point` fields and groups points into buckets that represent
```